sim/team-validator.ts

```diff
@@ -487,19 +493,20 @@ export class Validator {
 			(format.requirePlus || format.requirePentagon)) {
 			problems.push(`${name} has Secret Sword, which is only compatible with Keldeo-Ordinary obtained from Gen 5.`);
 		}
-		if (!lsetData.sources && lsetData.sourcesBefore <= 3 && dex.getAbility(set.ability).gen === 4 && !template.prevo && dex.gen <= 5) {
+		if (!lsetData.sources && lsetData.sourcesBefore <= 3 &&
+			dex.getAbility(set.ability).gen === 4 && !template.prevo && dex.gen <= 5) {
```

Comment:
I tend to strongly prefer intermediate variables, when code is too long to fit on one line. It has the added benefit of the intermediate variable's name being self-documenting.

```suggestion
		const hasGen3moves = !lsetData.sources && lsetData.sourcesBefore <= 3;
		if (hasGen3moves && dex.getAbility(set.ability).gen === 4 && !template.prevo && dex.gen <= 5) {
```